package.json

```diff
@@ -27,11 +27,13 @@
     "i18next": "^22.0.4",
     "i18next-browser-languagedetector": "^7.0.0",
     "javascript-time-ago": "^2.5.9",
+    "jszip": "^3.10.1",
     "lodash.debounce": "^4.0.8",
     "lodash.get": "^4.4.2",
     "lodash.isequal": "^4.5.0",
     "lodash.uniqueid": "^4.0.1",
     "md-editor-rt": "2.8.1",
+    "objects-to-csv": "^1.3.6",
```

Comment:
do you use it?

Reply:
later will be

Reply:
we have xlsx. it can work with csv. it doesn't suit you?

Reply:
I will check later, for now it is hard to know